doc/languages-frameworks/agda.section.md

```diff
@@ -13,9 +13,10 @@ $ nix-env -iA agda
 ```
 
 To use agda with libraries, the `agda.withPackages` function can be used. This function either takes:
```

Comment:
```suggestion
To use Agda with libraries, the `agda.withPackages` function can be used. This function either takes:
```
Can we capitilise this while fixing stuff?

Reply:
Yep, done